### PR TITLE
Fix health status - 1.11

### PR DESCRIPTION
**Pull request by skonto**
Back port of #681.

### CODE REVIEW

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml [hunk 1]
```diff
@@ -94,7 +94,7 @@ data:
                }
             ],
             "thresholds": "",
-            "title": "Knative Status",
+            "title": "Knative Current Status",
```
Comment (rhuss) [inline knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml:97]: What other kind of status then the _Current_ Status could be possible ?

**Review by rhuss — COMMENTED** [review]
I can't really comment with good faith on this PR (except some labeling nit), as I'm lacking the insights how we are exposing metrics. 

However, I would feel better if there would be some test. Not part of this PR but I wonder how this could be tested and verified (at least the happy path)

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml (hunk 1 above)
Comment (skonto) [reply knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml:97]: It depends there is a time-line bellow.

Comment (skonto) [reply knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml:97]: @rhuss regarding tests is on my todo list. Started working on this here #686

**Review by rhuss — APPROVED** [review]
From a superficial view it looks sound to me. The only thing I'm not 100% sure is whether there can be a race between `metrics.Registry.MustRegister()` and the place where serving/eventing controller register itself within the gauge.

If this is safe (@skonto you know best), feel free to unhold to get the PR merged

/lgtm 
/hold
/approve

**Comment by skonto** [comment]
@markusthoemmes ^^ pls let me know if this is ok with you.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Comment by skonto** [comment]
infra `Unable to connect to the server: dial tcp 52.54.221.49:6443: i/o timeout`

**Pull request merged**